### PR TITLE
Request permission to modify and release tfs-plugin

### PR DESCRIPTION
**Pull request by jpricketMSFT**
# Description
Request publish permission for Team Foundation Server Plugin
Github repo: https://github.com/jenkinsci/tfs-plugin
@DavidStaheli - as a committer, can you please review this request

# Submitter checklist for changing permissions

### Always

- [X] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [X] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [X] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [X] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by DavidStaheli** [comment]
Approved!  Permission confirmed.

**Comment by jpricketMSFT** [comment]
Adding @jeffyoung since he is on the request as well.

File: permissions/component-tfs-sdk.yml [hunk 1]
```diff
@@ -6,3 +6,6 @@ developers:
 - "oli_at_jsi"
 - "yacaovsnc"
 - "dastahel"
+- "jpricketMSFT"
```
Comment (daniel-beck) [inline permissions/component-tfs-sdk.yml]: This is case sensitive, please use all lowercase letters.

**Comment by daniel-beck** [comment]
@jpricketMSFT Looks good. Do you need me to add you as committer? Doesn't look like you have commit access to the repo. CC @DavidStaheli

**Comment by jpricketMSFT** [comment]
@daniel-beck do I need to change jpricketMSFT to jpricketmsft? 
Yes, Jeff and I both will need commit access.
Thanks!

**Comment by DavidStaheli** [comment]
Yes, approved for commit and release access.  Thanks @daniel-beck.

**Comment by daniel-beck** [comment]
@jpricketMSFT 

> do I need to change jpricketMSFT to jpricketmsft?

Yes.

**Comment by daniel-beck** [comment]
@DavidStaheli I added them as committers; they may need to accept the invitation into the jenkinsci org via email or on https://github.com/jenkinsci/

**Comment by DavidStaheli** [comment]
Thank you, @daniel-beck!

**Comment by jpricketMSFT** [comment]
Thanks @daniel-beck ! 
We appreciate all the help :)

**Pull request merged**